ChangeLog

```diff
@@ -454,7 +453,7 @@
     recovery2text(), role2text(), task2text(), and text2role()
   + libpe_status: deprecate functions pe_find_node(), pe_pref(),
     pe_rsc_is_anon_clone(), pe_rsc_is_bundled(), pe_rsc_is_clone(),
-    pe_rsc_is_unique_clone(),
+    and pe_rsc_is_unique_clone()
```

Comment:
if we want to have nice future diffs on the ChangeLog when deprecating more then the previous was probably better. for making it look nice with an 'and' the ',' should probably go away.